scripts/dynamicmodal.js

```diff
@@ -0,0 +1,188 @@
+    imageLink: 'desk/SnapshootPortfolio7.png',
+    tags: ['HTML', 'CSS', 'Javascript'],
+    role: 'Back End Dev',
+    company: 'Canopy',
+    year: '2015',
+    sourceLink: '',
+    demoLink: '',
+  },
+];
+
+(() => {
+  projectSectionDom.innerHTML = '';
+  for (let i = 0; i < projectsArray.length; i += 1) {
+    let techlist = '';
+    for (let j = 0; j < projectsArray[i].tags.length; j += 1) {
+      techlist = `${techlist}<li class="tag">${projectsArray[i].tags[j]}</li>`;
+    }
+    projectSectionDom.innerHTML = `${projectSectionDom.innerHTML}
+    <ul class="${projectsArray[i].card}">
+    <li class="project">
+      <img src="${projectsArray[i].imageLink}" alt="Snapshot">
+        <div class="description">
+            <h2 class="title">${projectsArray[i].title}</h2>
+            <div class="info">
+                <p class="subTitle">${projectsArray[i].company}</p>
+                <ul class="infoContainer">
+                    <li class="role">${projectsArray[i].role}</li>
+                    <li class="year">${projectsArray[i].year}</li>
+                </ul>
+            </div>
+            <p class="projectDescription">
+                ${projectsArray[i].description}  
+            </p>
+            <div class="tags">
+                <ul class="tagsFlex">
+                  ${techlist}
+                </ul>
+            </div>
+            <button type="button" class="viewProject prj-view-btn">
+                See Project
+            </button>
+        </div>
+    </li>
+</ul>`;
+  }
+  const viewBtn = document.querySelectorAll('.prj-view-btn');
+  for (let i = 0; i < viewBtn.length; i += 1) {
+    viewBtn[i].addEventListener('click', (event) => {
+      const i = event.currentTarget.index;
+      nameDom.innerHTML = projectsArray[i].title;
+      roleNameModal.innerHTML = projectsArray[i].role;
+      yearDateModal.innerHTML = projectsArray[i].year;
+      descriptionDom.innerHTML = projectsArray[i].modaldesc;
+      projectImageLinkDom.src = projectsArray[i].imageLink;
+      sourceLinkDom.action = projectsArray[i].sourceLink;
+      demoLinkDom.action = projectsArray[i].demoLink;
+      technologiesDom.innerHTML = '';
+      for (let j = 0; j < projectsArray[i].tags.length; j += 1) {
+        technologiesDom.innerHTML = `${technologiesDom.innerHTML}<li class="tag">${projectsArray[i].tags[j]}</li>`;
+      }
+      popupModal.classList.remove('vanish');
+      popupModal.classList.add('show');
+      mobileMenu.classList.add('blur');
+      logo.classList.add('blur');
```

Comment:
- [ ] Please kindly ensure the popup modal looks very similar to the original design on [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). At the moment, the text and buttons are overflowing the modal. Kindly compare your design with that of the original design below.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/157838135-0a6dddc9-c25d-4a0d-85ac-14af19126c52.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/157838293-92494a2c-532e-447b-9334-d8b9046dac2e.png)